tox.ini

```diff
@@ -9,6 +9,7 @@ extras = test,chem,examples,simulation,onnx
 passenv = CI
 setenv =
     SMOKE_TEST = true
+    BAYBE_TEST_ENV = FULLTEST
```

Comment:
Why are the additions in this file necessary?

Reply:
The reason is because the import tests will only work in a `dev` environment, since otherwise packages like `xyzpy` are not available and will trigger errors that should not be part of the test. See beginning of `test_imports.py`:
```python
pytestmark = pytest.mark.skipif(
    os.environ.get("BAYBE_TEST_ENV") != "FULLTEST",
    reason="Only possible in FULLTEST environment.",
)
```

I already foresee a smarter way here which will also make all our little helpers like `_ONNX_INSTALLED` and the various try-except blocks obsolete, and which will then enable to run the tests also in a basic environment. But that belongs to another PR, where I can then theoretically remove these variables again. However, I think they also might become handy in other situations. Anyway, for now they are needed (unless you have a better idea?) and we can decide later what to do with them 👍🏼